CHANGELOG.md

```diff
@@ -6,12 +6,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.0.0]
+### Added
+- Add client id header to GasFeeController ([#597](https://github.com/MetaMask/controllers/pull/597))
+
+### Changed
+- Improve transaction state management for custom networks ([#598](https://github.com/MetaMask/controllers/pull/598))
+- Make TokenRatesController support fiat conversion for more networks ([#585](https://github.com/MetaMask/controllers/pull/585))
+
+### Removed
+- **BREAKING:** Remove IsJsonable ([#496](https://github.com/MetaMask/controllers/pull/496))
```

Comment:
Should we say more about how consumers need to adapt to this change?

Reply:
Here's my suggestion:
```suggestion
- **BREAKING:** Simplify type of BaseControllerV2 state ([#496](https://github.com/MetaMask/controllers/pull/496))
  - Controllers based upon BaseControllerV2 might need to update their types.
  - Custom interfaces/classes will no longer be allowed in the controller state. Simple objects only.
```

Reply:
Updated